### DISCUSSION TITLE
Unitless defaults for line-heights

### DISCUSSION THREAD

**Discussion by davidwebca** [tailwindlabs/tailwindcss#4452] [Ideas]
Hello! I can't find anyone mentioning this so far, but maybe it's already been addressed.

I find myself resetting a lot the default line-heights because they're in rems instead of being unitless. I feel like line-heights and letter-spacings should always be relative to the font-size itself and it's already the case for [letter-spacing](https://tailwindcss.com/docs/letter-spacing) and some relative values are provided in [line-heights](https://tailwindcss.com/docs/line-height). but the defaults in the [defaultConfig.sub.js](https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js) are in rem as also shown in the docs. https://tailwindcss.com/docs/font-size. 

Didn't want to file a bug even tough it's arguable that it is one, but I was also wondering what lead to this decision and if there's already been discussions that lead to the decision.

https://codepen.io/davidwebca/pen/PopmQYE

**Comment by simonswiss** [comment]
Hey @davidwebca 👋

Here's a related discussion where I made an attempt to explain why we're setting `rem` values for the smaller font sizes' line-height values:

https://github.com/tailwindlabs/tailwindcss/discussions/3386#discussioncomment-295688

Quoting the relevant part:

> We didn't use unitless values for the smaller values, because it would be more complicated to come up with the math to hit a specific target rem value than using the actual rem value 😅

> The line-height affects elements height, and you want the element height to be a number that’s in the spacing scale. So we don’t use values like 1.5, because otherwise you'd get weird numbers hitting outside the scale.

> You could technically define all of these with numbers if you find out the exact values, nothing wrong with that.

Hopefully it makes sense 👍

**Reply by davidwebca** [reply]
Merci Simon! Totally had missed that conversation. It does make sense, but in my case I'll still need more often than not to remove the default values and change them to be unitless because the projects I'm working on often have custom selector with text sizes being specified without using tailwind's utilities. I think I'll just create myself a stub config that I'll require and extend in the meantime. 

If anyone else finds this and have the same use case, don't hesitate to comment to fuel the conversation.
